doc/api/fs.md

```diff
@@ -480,7 +480,7 @@ Reads data from the file and stores that in the given buffer.
 If the file is not modified concurrently, the end-of-file is reached when the
 number of bytes read is zero.
 
-#### `filehandle.readableWebStream([options])`
+#### `filehandle.readableWebStream()`
```

Comment:
If this change is acceptable, I need a maintainer's assistance to figure out what the proper way to signal this kind of "breaking change" to an experimental API is.  And I need to include a description of the change at least in the YAML below here.